src/restore.js

```diff
@@ -0,0 +1,43 @@
+const cache = require("@actions/cache");
+const core = require("@actions/core");
+const inputs = require("./inputs.js");
+const state = require("./state.js");
+const stackCache = require("./stack-cache.js");
+
+async function run() {
+  const stackYaml = inputs.getStackYaml();
+  const workingDirectory = inputs.getWorkingDirectory();
+  process.chdir(workingDirectory);
```

Comment:
So weird that this is blocking. But maybe not. I don't even know anymore.

Reply:
Yeah, lots of stuff I expect to be promises but aren't, and don't say `Sync` like the `fs` stuff. JS gonna JS.